flash/website/templates/home.html

```diff
@@ -3,63 +3,152 @@
+                        <div style="position: absolute; top: 40%; left: 50%; transform: translate(-50%, -50%); text-align: center; width: 100%;">
+                            <h4 style="margin: 0;">Box {{ box.box_number }}</h4>
+                            {% if forloop.last %}
+                                <form method="post" action="{% url 'delete_box' box.box_number %}" style="display: inline;">
+                                    {% csrf_token %}
+                                    <button type="submit" class="btn btn-danger btn-sm thumbnail delete-button">DELETE</button>
+                                </form>
+                            {% endif %}
+                        </div>
+                    </div>
+                    <div class="caption text-center">
+                        <a class="btn btn-mini btn-primary" href="{% url 'flashcard_program' box_number=box.box_number %}">» START</a>
+                    </div>
+                </div>
+            {% endfor %}
+            <div class="col-md-3" style="position: relative;">
+                <div class="thumbnail">
+                    <a href="#"><img src="{% static 'images/szablon.png' %}" alt="Create New Box"></a>
+                    <div style="position: absolute; top: 40%; left: 50%; transform: translate(-50%, -50%); text-align: center; width: 100%;">
+                        <h4 style="margin: 0;">Create New Box</h4>
+                    </div>
+                </div>
+                <div class="caption text-center">
+                    <form method="post" action="{% url 'create_new_box' %}">
+                        {% csrf_token %}
+                        <button type="submit" name="create_box" class="btn btn-mini btn-primary">CREATE NEW BOX</button>
+                    </form>
+                </div>
+            </div>
+            <div class="col-md-3" style="position: relative;">
+                <div class="thumbnail">
+                    <a href="#"><img src="{% static 'images/szablon.png' %}" alt="All cards"></a>
+                    <div style="position: absolute; top: 40%; left: 50%; transform: translate(-50%, -50%); text-align: center; width: 100%;">
+                        <h4 style="margin: 0;">All cards</h4>
+                    </div>
+                </div>
+                <div class="caption text-center">
+                    <a class="btn btn-mini btn-primary" href="{% url 'flashcard_program' box_number=0 %}">» START</a>
+                </div>
+            </div>
+        </div>
+    </div>
+  <div class="control-box">
+    <a data-slide="prev" href="#myCarousel" class="carousel-control left">‹</a>
+    <a data-slide="next" href="#myCarousel" class="carousel-control right">›</a>
   </div>
-  
-  <br/>
-  <br/>
-  <br/>
-  <br/>
-  <br/>
-  <br/>
-  <br/>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```

Comment:
I think jquery is already added in base.html, so probably no need to add again

Reply:
After deleting this, my JavaScript doesn't work properly.